README.md

```diff
@@ -321,6 +321,36 @@ and add them to the media library if their MIME types are supported. If the
 file/folder in _path_ does not exist/is not readable or is not provided then an
 error will be returned and the corresponding log message would be written into logs.
 
+## Internal Audio & Video Recording
+
+REQUIRED STEPS to activate recording:
+
+```bash
+adb shell pm grant io.appium.settings android.permission.RECORD_AUDIO
+adb shell pm grant io.appium.settings android.permission.WRITE_EXTERNAL_STORAGE
+adb shell appops set io.appium.settings PROJECT_MEDIA allow
+```
+
+Start Recording:
+```bash
+adb shell am start -n "io.appium.settings/io.appium.settings.Settings" -a io.appium.settings.recording.ACTION_START --es "recording_filename" "abc.mp4" --es "recording_priority" 0 --es "recording_max_duration" 900
+```
+
+### Arguments (see above start command as an example for giving arguments)
+- recording_filename (Mandatory) - You can type recording video file name as you want, but recording currently supports only "mp4" format so your filename must end with ".mp4"
+- recording_priority (Optional) - Default value: 0 which means recording thread priority is Maximum hovewer if you face performance drops during testing with recording enabled, you can reduce recording priority to 2 - Normal or 1 - Minimum
```

Comment:
shouldn't 1 be normal and 2 minimum if 0 is the high priority?

Reply:
done https://github.com/sirmordred/io.appium.settings/commit/2c8878a8387a0b425117abfd6db12a12292b25a0 yeah also to be consistent with the docs (...def is max, you can reduce priority...) i changed max(default) -> 2, normal -> 1 and min -> 0